bindings/bind_py_structure_manager.cc

```diff
@@ -880,32 +880,35 @@ namespace rascal {
     py::class_<ClusterRefBase>(m_internal, "ClusterRefBase");
     bind_cluster_refs(m_internal);
 
+    bind_atomic_structure(m_nl);
+
     py::module m_strc_mng = m_nl.def_submodule("StructureManager");
     m_strc_mng.doc() = "Structure Manager Classes";
     py::module m_adp = m_nl.def_submodule("Adaptor");
     m_adp.doc() = "Adaptor Classes";
 
-    using Manager_t = StructureManagerCenters;
-    bind_structure_manager<Manager_t>(m_strc_mng, m_internal);
-
-    // bind the factory function
-    bind_make_structure_manager<Manager_t>(m_nl);
+    // list of already binded manager stacks to avoid double binding
     std::set<std::string> name_list{};
-    BindAdaptorStack<Manager_t, AdaptorNeighbourList, AdaptorStrict>
-        adaptor_stack_1{m_nl, m_adp, m_internal, name_list};
-
-    BindAdaptorStack<Manager_t, AdaptorNeighbourList, AdaptorCenterContribution,
+    /*-------------------- struc-man-bind-start --------------------*/
+    // bind the root structure manager
+    bind_structure_manager<StructureManagerCenters>(m_strc_mng, m_internal);
```

Comment:
Why is the code updated too? Is this required for the doc changes ?

Reply:
I reshuffled a bit the code so that it becomes more meaningful when displayed in the doc.

Reply:
:+1: